docs/developer/extending/apps/synchronous-webhooks.mdx

```diff
@@ -0,0 +1,200 @@
+
+## Payment webhooks
+
+Payment webhooks are synchronous webhooks that allow delegating payment handling to Saleor Apps. Synchronous means that these webhooks expect a response of a particular shape to be returned from the app to continue processing in Saleor. Payment webhooks only support HTTP(S) protocol; they are sent as POST requests with `application/json` body and expect a response of the same content type.
+
+Available payment events are:
+
+| Name                        | Description                                                                                                                                                                                                             |
+| --------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `PAYMENT_AUTHORIZE`         | Authorize payment                                                                                                                                                                                                       |
+| `PAYMENT_CAPTURE`           | Capture payment                                                                                                                                                                                                         |
+| `PAYMENT_CONFIRM`           | Confirm payment                                                                                                                                                                                                          |
+| `PAYMENT_LIST_GATEWAYS`     | List payment gateways                                                                                                                                                                                                   |
+| `PAYMENT_PROCESS`           | Process payment                                                                                                                                                                                                         |
+| `PAYMENT_REFUND`            | Refund payment                                                                                                                                                                                                          |
+| `PAYMENT_VOID`              | Void payment                                                                                                                                                                                                            |
+
+To create an app that reacts to payment webhook, first you need to create a regular app as described in [this dashboard manual](/dashboard/apps.md). Make sure to make the app active and grant it the `HANDLE_PAYMENTS` permission.
```

Comment:
I would also mention here, that this is only the one way to do that. Someone can always create Thirdparty App, which could handle the payment actions.

Reply:
@koradon Did you meant "*not* the one way"?

Reply:
Sorry. Yes, I meant that this description suggests that you can build a payment app *only * by using local app.